### PR TITLE
Fix normative language in schema

### PR DESCRIPTION
**Pull request by jpmckinney**
Addresses #830

* [x] Includes #831 and #852, so merge those first. Diff from #852 is `https://github.com/open-contracting/standard/compare/normativeschema...normativeschema2`

No surprises here. Please check the Schema sheet of the spreadsheet for any rows where "Status" has "Disagree" (you can use the filters in the header row, or manually search the spreadsheet).

### CODE REVIEW

**Comment by jpmckinney** [comment]
@duncandewhurst WIP, so no review yet, but I noticed that codelist fields are inconsistent in saying things like:

* This is a closed codelist. Local XYZ should be mapped to this list. (2x)
* This is an open codelist. Local XYZ can be included in this list.
* Milestone codes is an open codelist, and codes should be agreed among data producers and the applications using that data.
* Values from the provided codelist should be used wherever possible, though extended values can be provided if the codelist does not have a relevant code. (2x)

I think it's better to omit such information, and instead ensure that the codelists documentation is clear on the rules around open and closed codelists (perhaps borrowing the improved language from OC4IDS). What do you think? It will save cleaning up the normative keywords in these clauses, which are mostly wrong ("should" instead of "must").

**Comment by duncandewhurst** [comment]
Yes, this makes sense to me.

I think we should state whether they are open or closed in the schema descriptions but keep the explanations of what that means in the docs.

**Comment by jpmckinney** [comment]
@duncandewhurst Ready for review.

File: standard/schema/release-schema.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
   "definitions": {
     "Planning": {
       "title": "Planning",
-      "description": "Information from the planning phase of the contracting process. Note that many other fields may be filled in a planning release, in the appropriate fields in other schema sections, these would likely be estimates at this stage e.g. totalValue in tender",
+      "description": "Information from the planning phase of the contracting process. Note that many other fields can be filled in a planning release, in the appropriate fields in other schema sections; these would likely be estimates at this stage, e.g. totalValue in tender.",
```
Comment (duncandewhurst) [inline standard/schema/release-schema.json]: `totalValue` doesn't exist, the field should be `value`

**Review by duncandewhurst — APPROVED** [review]
PR looks good, I left one comment on an error we should correct and noted one other thing:

The [related process](http://standard.open-contracting.org/1.1-dev/en/schema/codelists/#related-process) section of the codelist reference still mentions PQQs:

> The related process block is used at the release level to point backwards to prior processes, such as planning, PQQ or framework establishment...

This PR removes the mention of pre-qualifications from the related process schema, so we should remove it from the reference page too.

Neither are blockers, so I've approved the PR

**Comment by jpmckinney** [comment]
Good catches! I made the two changes and will merge.

**Pull request merged**